test/integration/scene-bans/add-scene-ban-handler.spec.ts

```diff
@@ -0,0 +1,333 @@
+import { test } from '../../components'
+import { makeRequest, owner, admin, nonOwner } from '../../utils'
+import { TestCleanup } from '../../db-cleanup'
+import * as handlersUtils from '../../../src/logic/utils'
+import { PlaceAttributes } from '../../../src/types/places.type'
+import { AuthLinkType } from '@dcl/crypto'
+import { InvalidRequestError, UnauthorizedError } from '../../../src/types/errors'
+
+test('POST /scene-bans - adds ban for a user from a scene', ({ components, stubComponents }) => {
+  const testPlaceId = `place-id-ban`
+
+  type Metadata = {
+    identity: string
+    parcel: string
+    sceneId: string
+    realm: {
+      serverName: string
+      hostname: string
+      protocol: string
+    }
+  }
+  let cleanup: TestCleanup
+  let metadataLand: Metadata
+  let metadataWorld: Metadata
+
+  beforeAll(async () => {
+    cleanup = new TestCleanup(components.database)
+  })
+
+  beforeEach(async () => {
+    metadataLand = {
+      identity: owner.authChain[0].payload,
+      parcel: '-9,-9',
+      sceneId: 'test-scene',
+      realm: {
+        serverName: 'test-realm',
+        hostname: 'https://peer.decentraland.zone',
+        protocol: 'https'
+      }
+    }
+
+    metadataWorld = {
+      identity: owner.authChain[0].payload,
+      parcel: '20,20',
+      sceneId: 'test-scene',
+      realm: {
+        serverName: 'name.dcl.eth',
+        hostname: 'https://worlds-content-server.decentraland.org/',
+        protocol: 'https'
+      }
+    }
+
+    jest.spyOn(handlersUtils, 'validate').mockResolvedValue({
+      ...metadataLand,
+      sceneId: 'test-scene',
+      isWorlds: false
+    })
+
+    stubComponents.places.getPlaceByParcel.resolves({
+      positions: [metadataLand.parcel],
+      id: testPlaceId,
+      world: false
+    } as PlaceAttributes)
+
+    stubComponents.places.getPlaceByWorldName.resolves({
+      id: testPlaceId,
+      world_name: 'name.dcl.eth',
+      world: true
+    } as PlaceAttributes)
+
+    stubComponents.lands.getLandPermissions.resolves({
+      owner: true,
+      operator: false,
+      updateOperator: false,
+      updateManager: false,
+      approvedForAll: false
+    })
+    stubComponents.sceneManager.getUserScenePermissions.resolves({
+      owner: false,
+      admin: false,
+      hasExtendedPermissions: false,
+      hasLandLease: false
+    })
+    stubComponents.worlds.hasWorldOwnerPermission.resolves(false)
+    stubComponents.worlds.hasWorldStreamingPermission.resolves(false)
+    stubComponents.worlds.hasWorldDeployPermission.resolves(false)
+    stubComponents.sceneAdminManager.isAdmin.resolves(false)
+    // Permission validation is now handled in the logic component
+    stubComponents.sceneBans.addSceneBan.resolves()
+  })
+
+  afterEach(async () => {
+    await cleanup.cleanup()
+    jest.restoreAllMocks()
+  })
+
+  describe('when user has valid permissions', () => {
+    describe('and user is scene owner', () => {
```

Comment:
Following a previous comment, the `addSceneBan` method is mocked. Taking this into consideration implies that we're only testing in this file the integration between the components and the HTTP layer, meaning that all of the other components are not called.
Maybe I didn't understand the purpose of the test, but, did you want to test the integration of the whole flow or just the component->HTTP integration?